assets/src/admin/theme-install/view/theme.js

```diff
@@ -72,10 +72,14 @@ export default wpThemeView.extend( {
 		}
 
 		if ( slug && ! this.isWPORGTheme( slug ) ) {
+			const externalLinkIcon = document.createElement( 'span' );
+			externalLinkIcon.classList.add( 'dashicons' );
+			externalLinkIcon.classList.add( 'dashicons-external' );
```

Comment:
NIT: This can be combined to a single call:

```suggestion
			externalLinkIcon.classList.add( 'dashicons', 'dashicons-external' );
```